### DISCUSSION TITLE
Using Summer 2 to upgrade pickaxe to Iridium

### DISCUSSION THREAD

**Discussion by waynn** [nathan-alden-sr/stardew-valley#44] [Ideas]
If you have the G and bars, I would argue it's more ideal to upgrade on Summer 2 so you only have one day without. 

@nathan-alden-sr

**Comment by nathan-alden-sr** [comment]
I had considered experimenting with upgrading the Pickaxe earlier as it's a real pain doing Skull Cavern without the Iridium Pickaxe. If you have the Bars and money, sure, go for it.
